elements/pfe-tabs/src/polyfills--pfe-tabs.js

```diff
@@ -1,51 +1,3 @@
-// @POLYFILL  Array.prototype.find
```

Comment:
pfe-tabs.js is using find() in two locations.  Do you think we still need the polyfill?

- https://github.com/patternfly/patternfly-elements/blob/7c25ac067d6bd71ed019a9c7fd67945d2c9208ea/elements/pfe-tabs/src/pfe-tabs.js#L398
- https://github.com/patternfly/patternfly-elements/blob/7c25ac067d6bd71ed019a9c7fd67945d2c9208ea/elements/pfe-tabs/src/pfe-tabs.js#L445

Reply:
nvm.  I see that you added it to Pfelement. 👍